share/module_check_a_mundo.F

```diff
@@ -269,6 +269,73 @@ SUBROUTINE check_nml_consistency
          count_fatal_error = count_fatal_error + 1
       END IF
 
+!-----------------------------------------------------------------------
+! Check that the dimensions are correctly secified when sf_urban_physics = 2 and 3
+!-----------------------------------------------------------------------
+      DO i = 1, model_config_rec % max_dom
+         IF ( model_config_rec % sf_urban_physics(i) == bepscheme .OR. &
+              model_config_rec % sf_urban_physics(i) == bep_bemscheme )  THEN
+           IF ( model_config_rec % num_urban_ndm .NE. 2 ) THEN
+             wrf_err_message = '--- ERROR: num_urban_ndm is wrong'
+             CALL wrf_message ( wrf_err_message )
+             wrf_err_message = '--- Please refer to example.namelist for correct setting of num_urban_ndm'
```

Comment:
@smileMchen @kkeene44 
Ming,
1. Add an "s"
2. Add a "the" 
example.namelist for   -->
examples.namelist for the

Kelly won't like the "the".  
Apparently, I don't either.
https://www.youtube.com/watch?v=ustXRPke9lM